src/metabase/db/jdbc_protocols.clj

```diff
@@ -62,7 +64,12 @@
   ;; Similarly, none of them handle ZonedDateTime out of the box either, so convert it to an OffsetDateTime first
   ZonedDateTime
   (set-parameter [t stmt i]
-    (jdbc/set-parameter (t/offset-date-time t) stmt i)))
+    (jdbc/set-parameter (t/offset-date-time t) stmt i))
+
+  ;; JDBC drivers don't know about Clojure ratios. So just set them as a double instead. That should be ok enough for now.
+  clojure.lang.Ratio
+  (set-parameter [ratio stmt i]
+    (jdbc/set-parameter (double ratio) stmt i)))
```

Comment:
idk why this is here when it was already merged into master as part of the part 1 PR